bots/zephyr_mirror_backend.py

```diff
@@ -596,13 +622,14 @@ def zcrypt_encrypt_content(zephyr_class, instance, content):
     return encrypted
 
 def forward_to_zephyr(message):
+    # type: (Dict[str, Any]) -> Any
```

Comment:
I think we can get more specific than `Any` for the return type here -- where in the function are the returns and what type are they returning?